src/test/java/ru/training/at/hw3/utils/webdriver/DriverConfigurator.java

```diff
@@ -0,0 +1,25 @@
+package ru.training.at.hw3.utils.webdriver;
+
+import io.github.bonigarcia.wdm.WebDriverManager;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import java.util.concurrent.TimeUnit;
+
+public class DriverConfigurator {
```

Comment:
Do you try implement **Singleton** design pattern here?

Reply:
No, I had no intentions to implement this pattern. The task requires to "correctly manage the place where you will create a driver instance", so I just studied the code from the lecture and decided to do something similar in my homework.